runtime/protocol/http-client/common/test/aws/smithy/kotlin/runtime/http/interceptors/FlexibleChecksumsResponseInterceptorTest.kt

```diff
@@ -163,29 +168,4 @@ class FlexibleChecksumsResponseInterceptorTest {
 
         op.roundTrip(client, TestInput("input"))
     }
-
-    @Test
-    fun testSkipsValidationWhenDisabled() = runTest {
-        val req = HttpRequestBuilder()
-        val op = newTestOperation<TestInput>(req)
-
-        op.interceptors.add(
-            FlexibleChecksumsResponseInterceptor<TestInput> {
-                false
-            },
-        )
-
-        val responseChecksumHeaderName = "x-amz-checksum-crc32"
-
-        val responseHeaders = Headers {
-            append(responseChecksumHeaderName, "incorrect-checksum-would-throw-if-validated")
-        }
-
-        val client = getMockClient(response, responseHeaders)
-
-        val output = op.roundTrip(client, TestInput("input"))
-        output.body.readAll()
-
-        assertNull(op.context.getOrNull(ChecksumHeaderValidated))
-    }
```

Comment:
**Question**: Why did we delete this test?

Reply:
This should be covered under ChecksumConfigTest > ResponseChecksumValidation but I added it back